addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3808,26 +3810,26 @@ var SnippetsMenu = Widget.extend({
      * If in the meantime the user has started to drag the snippet, it won't be
      * shown.
      *
+     * TODO: remove delay param in master
```

Comment:
Maybe one can argue about compatibility/custom code:
- the now ignored delay param (could have kept it in the code and make the caller specify 0 instead)
- the now gone `__showSnippetTooltip`

Seems reasonable as-is for me, given the private internal & niche methods.